src/modules/Dropdown/DropdownItem.js

```diff
@@ -75,7 +75,10 @@ DropdownItem.propTypes = {
   selected: PropTypes.bool,
 
   /** Display text. */
-  text: PropTypes.string,
+  text: PropTypes.oneOfType([
```

Comment:
I changed this to reflect the PropType definition for `options.text` in `Dropdown`. I can change back if specifying a string would be preferable.